src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
> hm this would be true if we had a || operator, but we have &&, no?

Where ? the last one ?
If yes, let's me explain by an example:
```
true && "Something"
> "Something"

false && "Something"
> false
```

But
```
true || "Something"
> true

false || "Something"
> "Something"
```

The second case is not the behavior the one who implemented the `findLastActive` method attends to get from it.